### PR TITLE
Add info for adding a price feed repo

### PR DESCRIPTION
**Pull request by dasanra**
First version on how to add a price feed

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -48,12 +48,13 @@ check out the :download:`DutchX Documentation <./_static/docs/DutchX_Documentati
    use-dutchx-as-an-oracle
    Add a token pair <add-token-pair>
    Run your own bots <run-your-own-bots>
+   Add a price feed for the bots <add-price-feed>
```
Comment (anxolin) [inline source/index.rst:51]: I would also add a link in the `run-your-own-bots` and in dx-examples-bots (both README.md and in the right position in the config file)

File: source/add-price-feed.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Add a price feed
```
Comment (anxolin) [inline source/add-price-feed.md]: I'll review it better in local, and maybe i'll create a commit on top of it. Thx Dani!

**Comment by anxolin** [comment]
I added some changes to the add price feed documentation

**Comment by dasanra** [comment]
Updated the reviewed version polishing some things.

For me it's ok now

**Pull request merged**